src/MongoDB/ReadPreference.c

```diff
@@ -398,6 +480,19 @@ static HashTable* php_phongo_readpreference_get_properties_hash(zval* object, bo
 		zend_hash_str_update(props, "maxStalenessSeconds", sizeof("maxStalenessSeconds") - 1, &z_max_ss);
 	}
 
+	if (!bson_empty0(hedge)) {
+		php_phongo_bson_state state;
+
+		PHONGO_BSON_INIT_DEBUG_STATE(state);
```

Comment:
This can also use `PHONGO_BSON_INIT_STATE`, like `php_phongo_javascript_get_properties_hash` does for `scope`.